### PR TITLE
[CloudPC] Added User\CloudPC Start and Stop methods

### PR DESCRIPTION
**Pull request by rbayetov**
Added Start and Stop methods under User\CloudPC

### CODE REVIEW

File: api-reference/beta/api/cloudpc-start.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+doc_type: apiPageType
+---
+
+# cloudPC: start
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Start specific Cloud PC. This is to cover Shift Work CloudPC end user scenarios.
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpc-start.md:16]: Start your own Cloud PC. Currently, only Shift Work Cloud PC is supported.

File: api-reference/beta/api/cloudpc-start.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+|Application|Not supported.|
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+To start the [cloudPC](../resources/cloudpc.md) of the specified user (who is the signed-in user) in the organization using delegated permission:
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpc-start.md:35]: using this API.

File: api-reference/beta/api/cloudpc-stop.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+doc_type: apiPageType
+---
+
+# cloudPC: stop
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Stop specific Cloud PC. This is to cover Shift Work CloudPC end user scenarios.
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpc-stop.md:16]: Stop your own Cloud PC. Currently, only Shift Work Cloud PC is supported.

File: api-reference/beta/api/cloudpc-stop.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+|Application|Not supported.|
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+To stop the [cloudPC](../resources/cloudpc.md) of the specified user (who is the signed-in user) in the organization using delegated permission:
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpc-stop.md:35]: using this API.

**Review by AshleyYangSZ — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/cloudpc-start.md (hunk 1 above)
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-start.md]: changed

File: api-reference/beta/api/cloudpc-stop.md (hunk 3 above)
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-stop.md]: changed

**Review by RuiHou105 — APPROVED** [review]

**Comment by JarbasHorst** [comment]
Hello @rbayetov, I reviewed your PR and left some comments. Please, take a look when you will have time.

File: api-reference/beta/api/cloudpc-stop.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+```http
+POST /me/cloudPCs/{cloudPCId}/stop
```
[the remark this answers is not in the document]
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-stop.md:38]: no. It does not. It only supports /me/cloudPC/... path

File: api-reference/beta/api/cloudpc-start.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+---
+title: "cloudPC: start"
+description: "Start a specific Cloud PC."
```
[the remark this answers is not in the document]
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-start.md]: updated

File: api-reference/beta/api/cloudpc-stop.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
+---
+title: "cloudPC: stop"
+description: "Stop a specific Cloud PC."
```
[the remark this answers is not in the document]
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-stop.md]: updated

**Comment by JarbasHorst** [comment]
Hi @Lauragra, this PR is ready for final review.

File: api-reference/beta/api/cloudpc-start.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+|Application|Not supported.|
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+To start the [cloudPC](../resources/cloudpc.md) of the specified user (who is the signed-in user) in the organization using delegated permission:
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpc-start.md:35]: Why do we have an introductory sentence here if there's only one syntax? Generally we add an introduction when we have more than one possible syntax.

File: api-reference/beta/api/cloudpc-start.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+doc_type: apiPageType
+---
+
+# cloudPC: start
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Start a specific Cloud PC for a user. Currently, only shift work Cloud PC is supported.
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpc-start.md:16]: Can you clarify, what is a "shift work Cloud PC"?

File: api-reference/beta/api/cloudpc-stop.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+doc_type: apiPageType
+---
+
+# cloudPC: stop
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Stop a specific Cloud PC for a user. Currently, only shift work Cloud PC is supported.
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpc-stop.md:16]: Please clarify what a shift work Cloud PC is here as well.

File: api-reference/beta/api/cloudpc-stop.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+|Application|Not supported.|
+
+## HTTP request
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+
+To stop the [cloudPC](../resources/cloudpc.md) of the specified user (who is the signed-in user) in the organization using delegated permission:
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpc-stop.md:35]: Not sure that we need an introductory sentence here.

Comment (rbayetov) [reply api-reference/beta/api/cloudpc-stop.md]: updated

File: api-reference/beta/api/cloudpc-start.md (hunk 9 above)
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-start.md]: Updated "shift work" (legacy name) to "Frontline" new branding and added link to resource to know more

File: api-reference/beta/api/cloudpc-stop.md (hunk 10 above)
Comment (rbayetov) [reply api-reference/beta/api/cloudpc-stop.md]: Updated "shift work" (legacy name) to "Frontline" new branding and added link to resource to know more

**Review by Lauragra — APPROVED** [review]

**Pull request merged**